contributors/devel/release/releasenotes.md

```diff
@@ -0,0 +1,94 @@
+# Creating PRs for release notes
+
+This document describes how to create PRs in Kubernetes repos so that the changes can be tracked with the
+Kubernetes release and published in release notes.
+
```

Comment:
Any thoughts on adding/linking to a few examples?  

1. An example of a user-visible change (something that might want a rn) and what a good rn looks like
1. An example a bugfix that the user likely won't care about and probably doesn't need a release note.